test/integration/galera_configuration/controls/galera_spec.rb

```diff
@@ -0,0 +1,52 @@
+include_dir = '/etc/mysql/conf.d'
+
+case os[:family]
+when 'centos', 'redhat'
+  include_dir = '/etc/my.cnf.d'
+end
+
+galera_config_file = "#{include_dir}/90-galera.cnf"
+
+control 'mariadb_galera_configuration' do
+  impact 1.0
+  title 'test global installation when galera configuration'
+
+  describe service('mysql') do
+    it { should be_enabled }
+    it { should be_running }
+  end
+
+  describe port('3306') do
+    it { should be_listening }
+  end
+end
+
+control 'mariadb_galera_configuration' do
+  impact 1.0
+  title "verify the rendered config in #{galera_config_file}"
+
+  content = <<-EOF.gsub(/^\s+/, '')
+    # DEPLOYED BY CHEF
+    [mysqld]
+    query_cache_size = 0
+    binlog_format = ROW
+    default_storage_engine = InnoDB
+    innodb_autoinc_lock_mode = 2
+    innodb_doublewrite = 1
+    server_id = 100
+    innodb_flush_log_at_trx_commit = 2
+    wsrep_on = ON
+    wsrep_provider_options = "gcache.size=512M"
+    wsrep_cluster_address = gcomm://
+    wsrep_cluster_name = galera_cluster
+    wsrep_sst_method = mariabackup
+    wsrep_sst_auth = sstuser:some_secret_password
+    wsrep_provider = /usr/lib64/galera/libgalera_smm.so
+    wsrep_slave_threads = 16
+    wsrep_node_address = 
+  EOF
+
+  describe file(galera_config_file) do
+    its('content') { should eq content }
+  end
```

Comment:
Can you please try using the [mysql_conf](https://docs.chef.io/inspec/resources/mysql_conf/) InSpec resource for testing each of these?

Reply:
Hey @ramereth,

I had considered using it but decided to match against the rendered file instead for two reasons:

* The `mysql_conf` resource doesn't check the server settings, it reads a file and checks the key/value pairs (so may as well compare the whole file)
* Not all options would be captured; additional options could be added unnoticed

If you still prefer me to switch I can do that.

Reply:
I've since come across [mysql_session](https://docs.chef.io/inspec/resources/mysql_session/), I can implement that in a future PR as the Galera cluster bit isn't being bootstrapped by Chef right now, currently it's a manual action on the first node.